index.js

```diff
@@ -5,12 +5,26 @@ const macos = () => execa.stdout('netstat', ['-anv', '-p', 'tcp'])
 	.then(data => Promise.all([data, execa.stdout('netstat', ['-anv', '-p', 'udp'])]))
 	.then(data => data.join('\n'));
 
-const linux = () => execa.stdout('netstat', ['-tunlp']);
+const linux = () => execa.stdout('ss', ['-tunlp']);
 const win32 = () => execa.stdout('netstat', ['-ano']);
 
 const getListFn = process.platform === 'darwin' ? macos : process.platform === 'linux' ? linux : win32;
-const cols = process.platform === 'darwin' ? [3, 8] : process.platform === 'linux' ? [3, 6] : [1, 4];
-const isProtocol = x => /^(tcp|udp)/i.test(x);
+const cols = process.platform === 'darwin' ? [3, 8] : process.platform === 'linux' ? [4, 6] : [1, 4];
+const isProtocol = x => /^\s*(tcp|udp)/i.test(x);
+
+const parsePid = input => {
+	if (input === undefined) {
+		return NaN;
+	}
+
+	const match = input.match(/(\d+)/);
```

Comment:
This is too simplistic for `ss`'s output. For instance, `users:(("python2",pid=25221,fd=13))` returns `2` as the pid.

How's this work for windows? `/(?:^|pid=)(\d+)/` It looks like it will work with [mac output](https://benohead.com/mac-netstat-list-ports-programs/).

Reply:
Thanks!  I can't believe I didn't notice that earlier.  Tests pass on both linux and windows.

Reply:
However, it appears to have broken Travis.